data/keymaps/i386/qwerty/sv-latin1.map

```diff
@@ -42,6 +42,7 @@ keycode   9 = eight            parenleft        bracketleft
 	alt     keycode   9 = Meta_eight
 keycode  10 = nine             parenright	bracketright
 	alt     keycode  10 = Meta_nine
+	control altgr keycode 10 = Control_bracketright
```

Comment:
> It's impossible to press [CTRL]+[]] with sv keyboard. Fix that.It's impossible to press [CTRL]+[]] with sv keyboard. Fix that.

It's not true.

```
$ git grep -w Control_bracketright data/keymaps/i386/qwerty/sv-latin1.map 
data/keymaps/i386/qwerty/sv-latin1.map:32:      control keycode   6 = Control_bracketright
data/keymaps/i386/qwerty/sv-latin1.map:61:      control keycode  27 = Control_bracketright
```

Reply:
I still have this question. You add `Control_bracketright` and say that it is impossible to press it. But `Control_bracketright` is already in this keymap.

What does this patch fix?

Reply:
> On [the Swedish layout](https://en.m.wikipedia.org/wiki/File:KB_Sweden.svg), both mapping are weird.

I used this to see the keyboard layout: http://kbdlayout.info/kbdsg and for Swiss German the `]` is on key code 27 and only for German it is on key code 10.

>  Therefore this change is valid, but it adds the 3rd way to press Control_bracketright.

Ok. I'm going to take this commit, but it seems to me that the commit message does not correspond to the changes. Can we describe the changes more clearly?

Reply:
sv-latin1.map: make Ctrl+AltGr+9 act as Ctrl+]